rsconnect_jupyter/__init__.py

```diff
@@ -56,9 +56,10 @@ def post(self, action):
             server_address = data['server_address']
             api_key = data['api_key']
             disable_tls_check = data['disable_tls_check']
+            cadata = data.get('cadata', None)
```

Comment:
At some point (not in this PR), we should consider some DRY cleanup on the parameters that are common to many operations.